README.md

```diff
@@ -5,12 +5,36 @@
 ### Introduction
 
 The service-catalog project is in incubation to bring integration with service
-brokers to the Kubernetes ecosystem via the
-[Open Service Broker API](https://github.com/openservicebrokerapi/servicebroker).
-A service broker is an endpoint that manages a set of services.  The end-goal of
-the service-catalog project is to provide a way for Kubernetes users to consume
-services from brokers and easily configure their applications to use those
-services.
+brokers to the Kubernetes ecosystem via the [Open Service Broker
+API](https://github.com/openservicebrokerapi/servicebroker). A service broker
+is an endpoint that manages a set of services.  The end-goal of the service-
```

Comment:
> The end-goal of the service-catalog project is to provide a way for Kubernetes users to consume services from brokers and easily configure their applications to use those services, without needing detailed knowledge about how those services are created /managed.

How about

The end-goal of the service-catalog project is to provide a tool for Kubernetes users easily configure their applications.  By having users applications consume services, the service broker provides an abstraction layer which allows for much easier application deployment and application dependency management.